Makefile

```diff
@@ -1,40 +1,28 @@
 PKG     = github.com/sapcc/swift-health-exporter
 PREFIX := /usr
 
-# NOTE: This repo uses Go modules, and uses a synthetic GOPATH at
-# $(CURDIR)/.gopath that is only used for the build cache. $GOPATH/src/ is
-# empty.
-GO            := GOPATH=$(CURDIR)/.gopath GOBIN=$(CURDIR)/build go
+GO            := GOBIN=$(CURDIR)/build go
```

Comment:
What's the motivation for removing GOPATH?

Reply:
There is no need for it. We can use the global modules at `$GOPATH/pkg/mod`.

Reply:
There is no need for it. We can use the specific modules from the global modules directory at `$GOPATH/pkg/mod`, instead of maintaining them locally in `.gopath/pkg/mod`.